vlm/src/main/scala/geotrellis/contrib/vlm/gdal/GDALBaseRasterSource.scala

```diff
@@ -114,8 +134,10 @@ trait GDALBaseRasterSource extends RasterSource {
     readBounds(bounds, 0 until bandCount)
   }
 
-  override def close = {
-    // baseDataset.delete()
-    dataset.delete()
-  }
+  override def close = dataset.delete()
+
+  /*override def finalize(): Unit = {
```

Comment:
why did this end up commented out? If not needed why not delete ?

Reply:
Should be removed!